### PR TITLE
[IMP] mail_tracking_mailgun: link to bounce event

### PR DESCRIPTION
**Pull request by chienandalu**
Forward port of #434

cc @Tecnativa

### CODE REVIEW

File: mail_tracking_mailgun/models/res_partner.py [hunk 1]
```diff
@@ -25,10 +25,13 @@ def _email_bounced_set(self, reason, event):
...
                 continue
-            body = _('Email has been bounced: %s\n'
-                     'Reason: %s\n'
-                     'Event: %s') % (partner.email, reason,
-                                     event and event.id or _('unknown'))
+            body = _(
+                'Email has been bounced: %s\n'
+                'Reason: %s\n'
+                'Event: '
+                '<a href=# data-oe-model=mail.tracking.event data-oe-id=%d>'
```
Comment (Tardo) [inline mail_tracking_mailgun/models/res_partner.py]:
```suggestion
                '<a href="#" data-oe-model="mail.tracking.event" data-oe-id="%d">'
```

**Review by Tardo — CHANGES_REQUESTED** [review]

File: mail_tracking_mailgun/models/res_partner.py (hunk 1 above)
Comment (chienandalu) [reply mail_tracking_mailgun/models/res_partner.py]: Applied in #434, but I'll amend with the changes made ther to avoid translation change

**Review by Tardo — APPROVED** [review]
LGTM :+1: Nice solution!

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This lacks the comment from previous version

**Comment by chienandalu** [comment]
Yes, I'm waiting for that one to be merged

**Comment by chienandalu** [comment]
There we go

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**